fenix/app/src/main/java/org/mozilla/fenix/compose/button/Button.kt

```diff
@@ -65,9 +73,10 @@ private fun Button(
 
         Text(
             text = text,
+            textAlign = TextAlign.Center,
             color = textColor,
             style = FirefoxTheme.typography.button,
-            maxLines = 1,
+            maxLines = if (enableAccessibilityDetection && fontScale > 1.0f) Int.MAX_VALUE else 2,
```

Comment:
wondering if we should hoist the maxLines param in the button's params and use that instead of hardcoded 2. Maybe there are callers who would want to set it to 1.

Reply:
I'd lean towards not making it a parameter (at least for now). There is no present need to be able to deviate from the default (now 2 lines), so I'd opt to not including it so there's no confusion as to whether to use this optional when these buttons are used in future screens by developers who don't have the context we're laying out here. If that need ever does arise, we'll hopefully be doing so via guidance from UX + design and it'll be a quick PR to make that change to the `Button` API.

Reply:
fair point, we can defer updating the api till we actually have a usecase. I guess gl is making the same point for `maxLines` below.